plenum/server/request_managers/write_request_manager.py

```diff
@@ -36,6 +37,8 @@ def __init__(self, database_manager: DatabaseManager):
         self.audit_b_handler = None
         self.future_primary_handler = None
         self.node_reg_handler = None
+        self.config = getConfig()
+        self._request_handlers_with_version = {}  # type: Dict[Tuple[int, str], List[WriteRequestHandler]]
```

Comment:
It makes sense to add a TODO comment to extend the version approach for all methods (not only `restore_state`) and have a common way to register req handlers (regardless of whether they are version or not).